.github/workflows/pypi.yml

```
@@ -23,7 +23,7 @@ jobs:
 
     - name: Install dependencies
       run: |
-        make poetry.config.native
+        make poetry.config.venv
```

No CI os pacotes precisam ser instalados de forma global

```suggestion
        make poetry.config.native
```